### PR TITLE
Update explore-htap.md, add tiflash recommendation

### PR DESCRIPTION
**Pull request by joey-yez**
### First-time contributors' checklist 

- [ ] I've signed [**Contributor License Agreement**](https://cla-assistant.io/pingcap/docs-cn) that's required for repo owners to accept my contribution.

### Which TiDB version(s) do your changes apply to? (Required)

**Tips for choosing the affected version(s):**

By default, **CHOOSE MASTER ONLY** so your changes will be applied to the next TiDB major or minor releases. If your PR involves a product feature behavior change or a compatibility change, **CHOOSE THE AFFECTED RELEASE BRANCH(ES) AND MASTER**.

For details, see [tips for choosing the affected versions (in Chinese)](https://github.com/pingcap/docs-cn/blob/master/CONTRIBUTING.md#版本选择指南).

- [*] master (the latest development version)
- [ ] v7.4 (TiDB 7.4 versions)
- [ ] v7.3 (TiDB 7.3 versions)
- [ ] v7.2 (TiDB 7.2 versions)
- [ ] v7.1 (TiDB 7.1 versions)
- [ ] v7.0 (TiDB 7.0 versions)
- [ ] v6.5 (TiDB 6.5 versions)
- [ ] v6.1 (TiDB 6.1 versions)
- [ ] v5.4 (TiDB 5.4 versions)
- [ ] v5.3 (TiDB 5.3 versions)
- [ ] v5.2 (TiDB 5.2 versions)
- [ ] v5.1 (TiDB 5.1 versions)
- [ ] v5.0 (TiDB 5.0 versions)

### What is the related PR or file link(s)?

- This PR is translated from:
- Other reference link(s):

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [ ] Need modification after applied to another branch 
- [ ] Might cause conflicts after applied to another branch

### CODE REVIEW

**Review by ayenller — APPROVED** [review]
LGTM

File: explore-htap.md [hunk 1]
```diff
@@ -35,6 +35,24 @@ TiDB HTAP 可以满足企业海量数据的增产需求、降低运维的风险
 
 如果想了解更多关于 TiDB HTAP 场景信息，请参阅 [PingCAP 官网中关于 HTAP 的博客](https://pingcap.com/zh/blog/?tag=HTAP)。
 
+当遇到以下技术挑战时，建议使用 TiDB HTAP 提升 TiDB 数据库整体表现：
```
Comment (Oreoxmt) [inline explore-htap.md:38]:
```suggestion
当遇到以下技术场景时，建议使用 TiDB HTAP 提升 TiDB 数据库整体表现：
```

File: explore-htap.md [hunk 2]
```diff
@@ -35,6 +35,24 @@ TiDB HTAP 可以满足企业海量数据的增产需求、降低运维的风险
...
+
+- 分析性能不足
+
+    某些查询属于复杂的数据分析，如聚合、关联等操作，而涉及的表无法使用索引或者索引的选择性很差，需要对大量数据（超过1000万行）进行分析查询，而 TiKV 引擎难以满足查询的性能需求。
```
Comment (Oreoxmt) [inline explore-htap.md:42]:
```suggestion
    某些查询属于复杂的数据分析，如聚合、关联等操作。在分析查询涉及大量数据（超过 1000 万行）时，如果这些查询涉及的表无法使用索引或者索引的选择性很差，而行存储引擎 [TiKV](/tikv-overview.md) 难以满足查询的性能需求。
```

File: explore-htap.md [hunk 3]
```diff
@@ -35,6 +35,24 @@ TiDB HTAP 可以满足企业海量数据的增产需求、降低运维的风险
...
+
+- 混合负载隔离
+
+    在高并发的 OLTP 业务同时，你可能还需要处理一些 OLAP 业务，同时需要避免这些 OLAP 查询干扰 OLTP 业务性能，确保系统的整体稳定性。
```
Comment (Oreoxmt) [inline explore-htap.md:46]:
```suggestion
    在高并发的 OLTP 业务同时，你可能需要处理一些 OLAP 业务，同时还需要避免 OLAP 查询影响 OLTP 业务性能，确保系统的整体稳定性。
```

**Review by Oreoxmt — APPROVED** [review]

**Review by flowbehappy — APPROVED** [review]

File: explore-htap.md [hunk 4]
```diff
@@ -35,6 +35,24 @@ TiDB HTAP 可以满足企业海量数据的增产需求、降低运维的风险
...
+
+    某些查询属于复杂的数据分析，如聚合、关联等操作。在分析查询涉及大量数据（超过 1000 万行）时，如果这些查询涉及的表无法使用索引或者索引的选择性很差，而行存储引擎 [TiKV](/tikv-overview.md) 难以满足查询的性能需求。
+
+- 混合负载隔离
+
+    在高并发的 OLTP 业务同时，你可能需要处理一些 OLAP 业务，同时还需要避免 OLAP 查询影响 OLTP 业务性能，确保系统的整体稳定性。
+
+- 简化 ETL 技术栈
+
+    当加工数据量规模中等（100 TB 以内）、数据加工调度流程不复杂、并发度不高（10 以内）时，希望简化替代 OLTP + ETL + OLAP 技术栈，使用一个数据库同时满足交易系统以及分析系统的需求，降低技术门槛和运维人员需求。
```
Comment (qiancai) [inline explore-htap.md:50]:
```suggestion
    当数据量为中等规模（100 TB 以内）、数据加工调度流程不复杂、并发度不高（10 以内）时，你可能希望简化技术栈，替换原本需要使用多个不同技术栈的 OLTP、ETL 和 OLAP 系统，使用一个数据库同时满足交易系统以及分析系统的需求，降低技术门槛和运维人员需求。
```

File: explore-htap.md [hunk 5]
```diff
@@ -35,6 +35,24 @@ TiDB HTAP 可以满足企业海量数据的增产需求、降低运维的风险
 
 如果想了解更多关于 TiDB HTAP 场景信息，请参阅 [PingCAP 官网中关于 HTAP 的博客](https://pingcap.com/zh/blog/?tag=HTAP)。
 
+当遇到以下技术场景时，建议使用 TiDB HTAP 提升 TiDB 数据库整体表现：
+
+- 分析性能不足
```
Comment (qiancai) [inline explore-htap.md:40]:
```suggestion
- 提升分析性能
```

File: explore-htap.md [hunk 6]
```diff
@@ -35,6 +35,24 @@ TiDB HTAP 可以满足企业海量数据的增产需求、降低运维的风险
 
 如果想了解更多关于 TiDB HTAP 场景信息，请参阅 [PingCAP 官网中关于 HTAP 的博客](https://pingcap.com/zh/blog/?tag=HTAP)。
 
+当遇到以下技术场景时，建议使用 TiDB HTAP 提升 TiDB 数据库整体表现：
```
[the remark this answers is not in the document]
Comment (qiancai) [reply explore-htap.md:38]: 因为前面已经提到的 3 种典型场景，所以这里要是再写场景，就有些混淆。

File: explore-htap.md (hunk 5 above)
Comment (qiancai) [reply explore-htap.md:40]: 其他 3 点都是正面需求，所以这一点也可以稍微对齐下

File: explore-htap.md (hunk 6 above)
[the remark this answers is not in the document]
Comment (joey-yez) [reply explore-htap.md:38]: 前面提到的是业务场景，这里是从技术的角度给出的场景，所以这里提到的是技术场景。

**Review by qiancai — APPROVED** [review]

**Pull request merged**